qemu/qemu.go

```diff
@@ -88,6 +88,9 @@ const (
 
 	// VirtioRng is the paravirtualized RNG device driver.
 	VirtioRng DeviceDriver = "virtio-rng"
+
+	// VirtioBalloon is the memory balloon device driver.
+	VirtioBalloon DeviceDriver = "virtio-balloon"
```

Comment:
good catch, nothing special, actually a typo that seems to work, on local VMs I am using `virtio-balloon-pci` not sure if there is something different.

Reply:
I think it will naturally fallback to `virtio-balloon-pci`. I guess that to allow different architectures to choose the right transport layer to emulate the device. Like ARM would use `virtio-mmio` and x86 would use `virtio-pci`.
Keep it this way, that's fine!

Reply:
too late  just updated.